client/src/App.js

```diff
@@ -1,23 +1,31 @@
-import React from "react";
+import React, { useState } from "react";
 import { BrowserRouter, Route, Switch, withRouter } from "react-router-dom";
 import "./App.css";
 import Nav from "./components/Nav/Nav";
 import Search from "./components/Search/Search";
 import List from "./components/List/List";
+// useContext 
+import {BusinessContext} from "./components/BusinessContext/BusinessContext";
 import About from "./components/About/About"
 import Team from "./components/Team/Team"
 import BizCard from "./components/BizCard/BizCard"
 
 function App() {
+
   return (
     <div className="App">
+      {/* useContext */}
+      <BusinessContext.Provider value="test: spacehop is the best">
```

Comment:
you should pass the state and setState method from above to the `value`
```suggestion
      <BusinessContext.Provider value={{ businesses, setBusinesses }}>
```

Reply:
I added the code!